packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1051,27 +1082,41 @@ class _LinesState extends State<Lines> with AutoDisposeMixin {
   Widget build(BuildContext context) {
     final pausedFrame = widget.selectedFrameNotifier?.value;
     final pausedLine = pausedFrame?.line;
+    // Used to account for the presence of `_ProfileInformationGutterHeader` at
+    // the top of the profiler gutter columns. Everything needs to be shifted
+    // down a single line so profiling information for line 1 isn't hidden by
+    // the header.
+    final profileInformationHeaderOffset =
+        widget.showProfileInformation ? 1 : 0;
 
     return SelectionArea(
       child: ListView.builder(
         controller: widget.scrollController,
         physics: const ClampingScrollPhysics(),
         itemExtent: CodeView.rowHeight,
-        itemCount: widget.lines.length,
+        itemCount: widget.lines.length + profileInformationHeaderOffset,
         itemBuilder: (context, index) {
-          final lineNum = index + 1;
+          if (widget.showProfileInformation && index == 0) {
+            return SizedBox(height: CodeView.rowHeight);
+          }
+          final lineNum = index - profileInformationHeaderOffset + 1;
```

Comment:
nit: create a local var to reduce duplication:
```
final dataIndex = index - profileInformationHeaderOffset;
final lineNumber = dataIndex + 1;
```

then below replace all uses of `index - profileInformationHeaderOffset` with `dataIndex`